SMLHelper/Patchers/CraftDataPatcher_Subnautica.cs

```diff
@@ -27,6 +27,8 @@ internal static void AddToCustomTechData(TechType techType, ITechData techData)
         private static void PatchForSubnautica(HarmonyInstance harmony)
         {
             PatchUtils.PatchClass(harmony);
+
+            Logger.Log("CraftDataPatcher is ready.", LogLevel.Debug);
```

Comment:
This is already called in [`CraftDataPatcher.cs#L88`](https://github.com/SubnauticaModding/SMLHelper/blob/EnumPatchingCleanup/SMLHelper/Patchers/CraftDataPatcher.cs#L88)


```suggestion
```